src/scene/graphics/shared/svg/SVGToGraphicsPath.ts

```diff
@@ -1,244 +1,202 @@
+/* eslint-disable no-console */
+import parse from 'parse-svg-path';
 import { warn } from '../../../../utils/logging/warn';
 
 import type { GraphicsPath } from '../path/GraphicsPath';
 
-const commandSizeMap = {
-    a: 7,
-    c: 6,
-    h: 1,
-    l: 2,
-    m: 2,
-    q: 4,
-    s: 4,
-    t: 2,
-    v: 1,
-    z: 0,
-};
+interface SubPath
+{
+    startX: number;
+    startY: number;
+}
 
 // TODO optimise and cache the paths?
 export function SVGToGraphicsPath(svgPath: string, path: GraphicsPath): GraphicsPath
 {
-    // get commands removing spaces at the end
```

Comment:
This whole section basically got replaced with the 3rd party lib which is more robust and tested.